src/css/code.css

```diff
@@ -21,11 +21,12 @@
 .doc .code-header .code-title {
   color: var(--color-grey-500);
   padding-right: 0.5em;
+  flex-grow: 1000;
 }
 
 .doc .code-header .code-language {
   flex-grow: 1;
-  padding: 0.6rem 0;
+  padding: 0.6rem 0.6rem 0 0;
```

Comment:
I'm not sure about setting the `flex-grow` value to 1000. The other option here would be to not have flex properties on here and add a `<div style="flex-grow:1">` in between.

Other than that it looks fine to me

Reply:
I know, it's hacky. I haven't been able to find a less-hacky solution though :/

I thought the `<div style="flex-grow:1">` in between could be a good idea, but unfortunately doesn't work - the center rightfully expands to take all the free space:
![Screenshot from 2022-11-09 15-43-56](https://user-images.githubusercontent.com/114478074/200860723-807ef2c5-1729-417c-bad1-46f26f6237ee.png)

Reply:
code.css:
```css
.doc .code-header .code-spacer {
  flex-grow: 1;
}

.doc .code-header .code-title {
  color: var(--color-grey-500);
  padding-right: 0.5em;
  flex-grow: 1;
}

.doc .code-header .code-language {
  flex-grow: 0; /* Don't grow the language element */
  padding: 0.6rem 0.6rem 0 0;
}
```
06-code.js
```js
    var originalTitle = div.parentNode.querySelector('.title')
    if (originalTitle) {
      var titleDiv = document.createElement('div')
      titleDiv.className = 'code-title'
      titleDiv.innerHTML = originalTitle.innerHTML

      originalTitle.style.display = 'none'

      children.push(titleDiv)
    }

    // Add spacer element
    children.push(createElement('div', 'code-spacer'))

    if (addCopyButton) {
```